content/blog/chif-driver-not-found.md

```diff
@@ -0,0 +1,121 @@
+---
+title: CHIF driver not found
+date: 2021-06-22T12:06:33.680Z
+author: François Donzé
+authorimage: https://gravatar.com/avatar/6f3982d1461459da475f47e3a6c89d1d?s=192
+tags:
+  - ilo-restful-api
+  - redfish
+  - chif driver
+  - ilorest
+  - ilo 5
+---
+## Introduction
+
+When using the HPE RESTful Interface Tool ([iLOrest](http://hpe.com/info/resttool)) or other applications, like the Smart Update Manager ([SUM](https://www.hpe.com/us/en/servers/smart-update.html)) or the integrated Smart Update Tool ([iSUT](https://support.hpe.com/hpesc/public/docDisplay?docLocale=en_US&docId=emr_na-a00068223en_us)), you may get errors similar to the one shown below without any other information to identify the origin of the problem or how to fix it.
+
+Chif driver not found, please check that the chif driver is installed
```

Comment:
Hi, Francois. Thank you for taking my comments into account. In the preview for this now, this particular line doesn't show up with the black background that makes it look like a code snippit. You might want to format it as such.